projects/storefrontlib/cms-components/myaccount/address-book/address-book.component.html

```diff
@@ -69,9 +73,13 @@
 
   <ng-container *ngIf="showEditAddressForm">
     <section>
-      <p class="cx-section-msg">
+      <!-- TODO:  for next major releasee remove below p tag and feature level check from the h2 tag-->
```

Comment:
Can we also create a Jira ticket for such removal, mark it for 6.0 and put ticket number to the comment for better reference?